Source/Core/TestCase.jsl

```diff
@@ -17,10 +17,15 @@ Define Class( "UtTestCaseFixture",
 	// Variable: name
 	// Name of the test case
 	name = "";
-	// Variable:setup expr
+	// Variable: setup expr
 	setup expr = Empty();
 	// Variable: teardown expr 
 	teardown expr = Empty();
+	// Variable: skip flag
+	skip expr = Expr(0);
```

Comment:
I can change it to just `0` or maybe `ut expr literal(0)`. We want it to be false so that the default value works well for the implicit test case created when you do `ut test("string", ...)`. I just used `Expr(0)` since I wanted to indicate that it was supposed to be an expression and that it is false by default.